src/NServiceBus.Core/Recoverability/IRecoverabilityActionContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NServiceBus.Pipeline
+{
+    using System.Collections.Generic;
+    using Transport;
+
+    /// <summary>
+    /// Provide context to behaviors on the recoverability action pipeline.
```

Comment:
```suggestion
    /// Provide context to recoverability actions.
```

Since the actions isn't really a pipeline?